spec/lib/mcb/courses_editor_cli_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'mcb_helper'
+
+describe MCB::CoursesEditorCLI do
+  def run_with_input_commands(*input_cmds)
+    stderr = nil
+    output = with_stubbed_stdout(stdin: input_cmds.join("\n"), stderr: stderr) do
+      yield
+    end
+    [output, stderr]
+  end
+
+  let(:provider) { nil } # not needed for these specs
+  subject { described_class.new(provider) }
```

Comment:
why do these two lines have to be here?

Reply:
They don't – I've condensed them into one.